superset-frontend/src/components/Select/Select.tsx

```diff
@@ -165,6 +165,11 @@ export interface SelectProps extends PickedSelectProps {
    * Will not apply to predefined `options` array when users are not searching.
    */
   sortComparator?: typeof DEFAULT_SORT_COMPARATOR;
+  /**
+   * Customize who the container node is.
+   * Parent node by default.
+   */
+  getPopupContainer?: (triggerNode?: any) => any;
```

Comment:
`getPopupContainer` comes from Ant Design. Can you include it in `PickedSelectProps` instead?

Reply:
If I do that, then the definition in `src/filters/components/Select/SelectFilterPlugin.tsx` will be invalid:
`(trigger: HTMLElement) => ParentNode | null' is not assignable to type 'RenderDOMFunc'`

And I didn't want to touch any other non-related component. But if you think I must, I will.

Reply:
I think it's important to preserve Ant Design's original signature. You can change `src/filters/components/Select/SelectFilterPlugin.tsx` to:

```
getPopupContainer={
  showOverflow
    ? () => (parentRef?.current as HTMLElement) || document.body
    : (trigger: HTMLElement) =>
        (trigger?.parentNode as HTMLElement) || document.body
}
```

Reply:
Updated